<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateManagerDialog</class>
 <widget class="QDialog" name="TemplateManagerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>933</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template Manager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="karenmeu.qrc">
    <normaloff>:/rc/images/templateManager.png</normaloff>:/rc/images/templateManager.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,1,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Templates</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
     <item>
      <widget class="QComboBox" name="cbTemplates"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddTemplate">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="karenmeu.qrc">
         <normaloff>:/rc/images/new.png</normaloff>:/rc/images/new.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveTemplate">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="karenmeu.qrc">
         <normaloff>:/rc/images/save.png</normaloff>:/rc/images/save.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveTemplate">
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="karenmeu.qrc">
         <normaloff>:/rc/images/remove.png</normaloff>:/rc/images/remove.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Template Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtTemplateName"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>File Name Rule</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edtFileNameRule"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Content Template</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MarkdownEditor4" name="edtContent" native="true"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MarkdownEditor4</class>
   <extends>QWidget</extends>
   <header>markdowneditor4.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="karenmeu.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TemplateManagerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TemplateManagerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
